english/ninja/ninja.md

```diff
@@ -0,0 +1,21 @@
+[../buy-gameboy/buy-gameboy.md](gameboy)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.